examples/master.js

```diff
@@ -122,8 +99,15 @@ async function startMaster(localView, remoteView, formValues, onStatsReport, onR
     master.signalingClient.on('sdpOffer', async (offer, remoteClientId) => {
         console.log('[MASTER] Received SDP offer from client: ' + remoteClientId);
 
+        // Get ICE server configuration.
+        const getIceServerConfigPromise = master.kinesisVideoSignalingChannelsClient
```

Comment:
I think we should await here, and pass the configuration directly to `RTCPeerConnection` instead of using `setConfiguration`.